usr/sbin/so-elastic-start

```diff
@@ -15,9 +15,6 @@ echo
 echo "Starting containers:"
 echo
 
-# FREQSERVER
```

Comment:
I noticed the removal of comments for the recent test package, therefore I continued with that approach.  If this is not desired, I can resubmit with the previous comments.

Reply:
I added the comments to my dev repo, but when I "backported" the fix to the "stable" repo, I didn't include the comments.  However, that was only because I was in a hurry and knew that the "stable" repo is only temporary and will be overwritten by the dev repo in a few weeks.

As a general rule, we should never remove comments.  Probably the only exception to that rule would be if the comment was incorrect and, even then, the incorrect comment should be immediately replaced with a correct comment.